docs/modules/advanced_speed_servo_info.rst

```diff
@@ -32,12 +16,12 @@ Supported IQUART Clients
         * :ref:`hobby_input`
         * :ref:`uavcan_node`
         * :ref:`coil_temperature_estimator`
-        * :ref:`power_safety`
+        * :ref:`power_safety_table`
         * :ref:`stow_user_interface`
         * :ref:`arming_handler`
         * :ref:`stopping_handler`
         * :ref:`iquart_flight_controller_interface`
-        * :ref:`throttle_source_manager` - v0.1.0 and above
+        * :ref:`throttle_source_manager`
```

Comment:
Ditching the note on the versioning here?

Reply:
Yea, I figured we've done enough releases since 0.1.0 that we could take it off. I don't feel strongly though, I'm content to add it back